lib/scrivener/paginater/ecto/query.ex

```diff
@@ -7,35 +7,39 @@ defimpl Scrivener.Paginater, for: Ecto.Query do
 
   @spec paginate(Ecto.Query.t, Scrivener.Config.t) :: Scrivener.Page.t
   def paginate(query, %Config{page_size: page_size, page_number: page_number, module: repo, caller: caller, options: options}) do
-    total_entries = Keyword.get_lazy(options, :total_entries, fn -> total_entries(query, repo, caller) end)
+    opts = [prefix: Keyword.get_lazy(options, :prefix, fn -> nil end)]
```

Comment:
I don't think this needs to be a `get_lazy`, considering we're passing it down inside another `get_lazy` call.